src/libvfcinstrument/libVFCInstrument.cpp

```diff
@@ -214,7 +259,7 @@ struct VfclibInst : public ModulePass {
   bool runOnFunction(Module &M, Function &F) {
     if (VfclibInstVerbose) {
       errs() << "In Function: ";
-      errs().write_escaped(F.getName()) << '\n';
+      errs().write_escaped(demangle(F.getName())) << '\n';
```

Comment:
This is great. Should we mention in documentation, that function names are demangled ?

Reply:
I don't think so since it's natural for the user to have demangled names

Reply:
Ok agreed.